readme-template.md

```diff
@@ -6,19 +6,96 @@ This is a library which allows you to set, get, and remove private and public da
 
 [Data Schema](./DATA-MODEL.md)
 
-[API Documentation](./API-SPECIFICATION.md)
-
-## Usage
-Simply install using npm
+## Installation
+Install 3box in your npm project:
 ```
 $ npm install 3box
 ```
-and then import into your project
+
+## Usage
+Import the 3box module
 ```js
 const ThreeBox = require('3box')
+```
+or use the dist build in your html code
+```js
+<script type="text/javascript" src="../dist/3box.js"></script>
+```
 
-ThreeBox.openBox(web3.eth.accounts[0]).then(threeBox => {
-  // Code goes here...
+### Get the public profile of an address
+Using `async/await`
+```js
+const profile = await ThreeBox.getProfile('0x12345abcde')
+console.log(profile)
+```
+or using `.then`
+```js
+ThreeBox.getProfile('0x12345abcde').then(profile => {
+  console.log(profile)
 })
 ```
 
+### Get, set, and remove data
+To get private data, or modify public or private data in a users 3box you first have to open it by calling the `openBox` method. This method prompts the user to authenticate your dapp and returns a promise with a threeBox instance. You can only set, get, and remove data of users that are currently interacting with your dapp. Below `web3provider` refers to the object that you would get from `web3.currentProvider`, or request directly from the web3 browser, e.g. MetaMask.
```

Comment:
"To get or modify data in a user's 3Box, first open their 3Box by calling the `opeBox` method."